lib/ansible/modules/cloud/docker/docker_stack.py

```diff
@@ -81,6 +81,9 @@
 requirements:
   - jsondiff
   - pyyaml
+
+notes:
+  - Return values C(out) and C(err) have been deprecated and will be removed. Use C(stdout) and C(stderr) instead.
```

Comment:
```suggestion
  - Return values I(out) and I(err) have been deprecated and will be removed in Ansible 2.14. Use I(stdout) and I(stderr) instead.
```
In general: `I(...)` is used for option names, `C(...)` for option values